target/linux/ath79/dts/qca9533_comfast_cf-e110n.dts

```diff
@@ -0,0 +1,160 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "comfast,cf-e110n", "qca,qca9533";
+	model = "COMFAST CF-E110N";
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &wan;
+		led-failsafe = &wan;
+		led-upgrade = &wan;
+	};
```

Comment:
You seem to have plenty of LEDs, I think, that it's better for UX if you could use different LED/colors to signal different states. You don't want user to turn off the CPE during upgrade, right?

Reply:
Good idea. Let's use the red rrsilow LED:

```
	aliases {
		serial0 = &uart;
		led-boot = &wan;
		led-failsafe = &wan;
		led-upgrade = &rssilow;
	};
```